x-pack/test/functional/apps/ml/data_frame_analytics/feature_importance.ts

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { DeepPartial } from '../../../../../plugins/ml/common/types/common';
+import { DataFrameAnalyticsConfig } from '../../../../../plugins/ml/public/application/data_frame_analytics/common';
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getService }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const ml = getService('ml');
+
+  describe('jobs cloning supported by UI form', function () {
```

Comment:
Looks like this has been copy and pasted from another test.

Reply:
Fixed here https://github.com/elastic/kibana/pull/82677/commits/3327bd5918cf9d46c81d370c96604b67328a64dc